src/components/Global/BaseDialog.vue

```diff
@@ -0,0 +1,70 @@
+<template>
+  <v-dialog v-model="show" persistent max-width="650" :scrollable="scrollable" elevation scrim="surface-1">
+    <v-card :id="id" class="dialog-card" color="surface-1">
+      <v-card-title class="d-flex justify-space-between pt-4">
+        <h1 v-if="title">{{ title }}</h1>
+        <slot name="title" />
+      </v-card-title>
+      <v-card-text>
+        <slot name="body" />
+      </v-card-text>
+      <div class="dialog-actions d-flex justify-end">
+        <slot name="actions" />
+      </div>
+    </v-card>
+  </v-dialog>
+</template>
+
+<script>
+export default {
+  name: 'BaseDialog',
+  emits: ['update:modelValue'],
+  props: {
+    modelValue: {
+      type: Boolean,
+      required: true,
+    },
+    title: {
+      type: String,
+      default: '',
+    },
+    id: {
+      type: String,
+      required: true,
+    },
+    scrollable: {
+      type: Boolean,
+      default: false,
+    },
```

Comment:
We could have used a generic attribute binding to set any vuetify props from the parent (this one is needed for the Three and Scrap dialogs since they have long lists of cards), but I figured making it more explicit would be preferable. It has the downside of requiring an update to this component if we want a consumer to make use of an additional vuetify dialog prop but at least the list of options will be clear when looking at this component itself. Probably make it easier to refactor it out of vuetify if we ever do that as well

Reply:
I think requiring an update to modify it is fine, we have a specific way to use this component so it doesn't need to be totally generic.